content/about/student_stories.rst

```diff
@@ -9,6 +9,55 @@ Our Student Stories are a special collection of experiences and thoughts from
 the students themselves. Check out what our student employees find exciting and
 worth taking some time to tell you about.
 
+Cody Holliday on the Department of Energy Cyber Defense Competition 2018
+------------------------------------------------------------------------
+4/23/18
```

Comment:
There needs to be an extra newline after the date

Reply:
@Codysseus this still needs to be fixed!

Reply:
Didn't you mean before ;P